### DISCUSSION TITLE
Frontend UI not enabled

### DISCUSSION THREAD

**Discussion by Streamnet-coder** [dwainscheeren/dwains-lovelace-dashboard#757] [Dwains Dashboard V3 Alpha]
I got stacked onto the YAML config as default therefore not allowing me to load UI enabled dashboard, when attempting to load the dashboard. How can I disable YAML config as default dashboard. Plus  frontend option is completely missing from my HACS

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
